spark/src/main/scala-spark-master/shims/DeltaTableValueFunctionsShims.scala

```diff
@@ -29,7 +29,7 @@ object DeltaTableValueFunctionsShims {
    * In Spark 4.0, SPARK-46331 [https://github.com/apache/spark/pull/44261] removed CodegenFallback
    * from a subset of DateTime expressions, making the `now()` expression unevaluable.
    */
-  def evaluateTimeOption(value: Expression): String = {
+  def evaluateOptionExpr(value: Expression): String = {
```

Comment:
Hm. Is this not still time-related? There is `ComputeCurrentTime` below

Reply:
you are right, we don't need to rename it.